script/fs-bench/work/Dockerfile

```diff
@@ -0,0 +1,52 @@
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+FROM golang:1.13
+
+# basic tools
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y libbtrfs-dev libseccomp-dev fuse python \
+            apt-transport-https software-properties-common && \
+    curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - && \
+    add-apt-repository \
+      "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" && \
+    apt-get update -y && \
+    apt-get install --no-install-recommends -y docker-ce-cli && \
+    curl -Lo gnuplot-5.2.8.tar.gz \
+    https://sourceforge.net/projects/gnuplot/files/gnuplot/5.2.8/gnuplot-5.2.8.tar.gz/download && \
```

Comment:
Why not apt get?

Reply:
Using apt-get, gnuplot reports `libqt5core.so.5: cannot open shared object file: no such file or directory`, even with libqt5core properly installed. fedora faces the same problem:
https://bugzilla.redhat.com/show_bug.cgi?id=1666427